### PR TITLE
changes in index.cyper and unit test data add and aws test case modify

### PR DESCRIPTION
**Pull request by sagarnavale**

### CODE REVIEW

File: cartography/cli.py [hunk 1]
```diff
@@ -566,7 +566,7 @@ def run_azure(request):
         graph_scope=request['azure']['graph_scope'],
         azure_scope=request['azure']['azure_scope'],
         params=request['params'],
-        neo4j_max_connection_lifetime=3600,
+        # neo4j_max_connection_lifetime=3600,
```
Comment (mpurusottamc) [inline cartography/cli.py:569]: we should not disable this parameter.

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
minor change needed

**Comment by mpurusottamc** [comment]
@ganeshjagdale9545 can you please review this PR?

File: cartography/cli.py (hunk 1 above)
Comment (ganeshjagdale9545) [inline cartography/cli.py:569]: Uncomments this

File: cartography/data/indexes.cypher [hunk 2]
```diff
@@ -107,6 +107,10 @@ CREATE INDEX ON :RDSSnapshot(id);
 CREATE INDEX ON :AWSESReservedInstance(id);
 CREATE INDEX ON :AWSSESIdentity(id);
 CREATE INDEX ON :GCPDNSZone(id);
+CREATE INDEX ON :GCPDNSPolicy(id);
+CREATE INDEX ON :GCPDNSKey(id);
+CREATE INDEX ON :GCPLabel(id);
+
```
Comment (ganeshjagdale9545) [inline cartography/data/indexes.cypher:113]: Remove this empty line

File: cartography/data/indexes.cypher [hunk 3]
```diff
@@ -317,3 +321,6 @@ CREATE INDEX ON :KubernetesContainer(name);
 CREATE INDEX ON :KubernetesContainer(image);
 CREATE INDEX ON :KubernetesService(id);
 CREATE INDEX ON :KubernetesService(name);
+CREATE INDEX ON :AzureFunction(id);
+CREATE INDEX ON :AzureLocation(id);
+
```
Comment (ganeshjagdale9545) [inline cartography/data/indexes.cypher:326]: Remove this empty line

File: tests/data/aws/route53.py [hunk 4]
```diff
@@ -37,6 +37,8 @@
     {
         "Id": "/hostedzone/FAKEZONEID1",
         "Name": "test.com.",
+        'arn': 'arn:aws:kms:eu-west-1:000000000000:alias/key2-cartography',
+        'consolelink': '',
```
Comment (ganeshjagdale9545) [inline tests/data/aws/route53.py:41]: Don’t use empty strings instead of use some string value

File: tests/data/aws/route53.py [hunk 5]
```diff
@@ -46,6 +48,8 @@
     {
         "Id": "/hostedzone/FAKEZONEID2",
         "Name": "test.com.",
+        'arn': 'arn:aws:kms:eu-west-1:000000000000:alias/key2-cartography',
+        'consolelink': '',
```
Comment (ganeshjagdale9545) [inline tests/data/aws/route53.py:52]: same as above

File: tests/integration/cartography/intel/aws/test_sqs.py [hunk 6]
```diff
@@ -79,4 +79,5 @@ def test_load_sqs_queues(neo4j_session, *args):
         )
         for r in relation
     }
+    print(actual_relationship)
```
Comment (ganeshjagdale9545) [inline tests/integration/cartography/intel/aws/test_sqs.py:82]: Remove this print statement

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
few minor changes needed

**Pull request merged**